### ISSUE TITLE
Split ENDIF IF in some cases

### ISSUE THREAD

**Issue by VictorKensing** [fib-at-isobar/beautyAmp#12]
I've found that in some cases is ENDIF and IF merged together:

example:

```
%%[
    IF @myVar == "" THEN
        SET @myVar = "hi"
        SET @myVar2 = "hi"
    ELSE
        SET @myVar = "hi2"
        /*comment*/
    ENDIF
    IF @myVar2 == "" THEN
        SET @myVar2 = "hello"
    ENDIF
]%%
```

ENDIFIF is after formatting:

```
%%[
    IF @myVar == "" THEN
        SET @myVar = "hi"
        SET @myVar2 = "hi"
    ELSE
        SET @myVar = "hi2"
        /*comment*/
        ENDIFIF @myVar2 == "" THEN
        SET @myVar2 = "hello"ENDIF
]%%
```

**Comment by fib-at-isobar** [comment]
Hello @majoabraham,
I will take a look when possible.
Thanks for pointing it out!
Regards,
Filip

**Comment by fib-at-isobar** [comment]
Hello @majoabraham,
Seems I have found the issue. I am currently testing this fix, and will release it soon.

**Comment by fib-at-isobar** [comment]
Hello, @majoabraham, it's fixed with new version 0.4.0 - enjoy!
